packages/editor/src/store/selectors.js

```diff
@@ -1749,18 +1749,6 @@ export function isPublishingPost( state ) {
 	return !! stateBeforeRequest && ! isCurrentPostPublished( stateBeforeRequest );
 }
 
-/**
- * Returns the provisional block client ID, or null if there is no provisional
- * block.
- *
- * @param {Object} state Editor state.
- *
- * @return {?string} Provisional block client ID, if set.
- */
-export function getProvisionalBlockClientId( state ) {
```

Comment:
I hesitated to keep this selector fo BC but the probability for it to be used outside Gutenberg is 0 IMO

Reply:
Alternatively, we could keep the function, and have it return `null` (the default reducer state) so it would at least not throw an error and be as close as possible to a real value with the state no longer existing.

But I'd probably agree it's unlikely that this is used.